✨(react) add monoline props to multi select (#205)
openfun/cunningham: packages/react/src/components/Forms/Select/multi.mdx

  <Story id="components-forms-select-multi--hidden-label"/>
</Canvas>

## Monoline

You can use `monoline` props in order to make sure the component will never wrap on multiple lines.

> When using this mode the selected options will keep being displayed in the menu with checkboxes in order to make sure the user can

Reviewer: nit:
```suggestion
When using this mode the selected options will be kept in the menu with checkboxes in order to make sure the user can
```
